pom.xml

```diff
@@ -142,5 +148,10 @@
       <version>${junitVersion}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-simple</artifactId>
+      <version>1.6.1</version>
+    </dependency>
```

Comment:
```suggestion
      <version>1.7.32</version>
      <scope>test</scope>
    </dependency>
```

Reply:
o sorry. thanks! I just did the same thing. Maybe I should just wait until you fix everything :-)

Reply:
Would be great to pin the latest stable version of SLF4J (1.7.32).

Even better would be defining the version in a property (e.g., `slf4jVersion`) similar as jUnit version etcetera are defined in the `pom.xml`.

Reply:
done

Reply:
still running the tests

Reply:
io.usethesource.capsule.SpecializedMapPropertiesTest takes a long time to complete. do we generate to many different parameters there or is this necessary to get good coverage?

Reply:
> io.usethesource.capsule.SpecializedMapPropertiesTest takes a long time to complete. do we generate to many different parameters there or is this necessary to get good coverage?

I’ll follow up with you offline on that subject.